.github/workflows/build_wheels.yml

```diff
@@ -69,7 +74,7 @@ jobs:
           CIBW_BEFORE_BUILD: pip install cython
           CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.manylinux_image }}
           CIBW_ARCHS_LINUX: ${{ matrix.cibw_archs }}
-          CIBW_BUILD: ${{ github.ref == 'refs/heads/master' && 'cp39-* cp310-*' || 'cp310-*' }}
+          CIBW_BUILD: ${{ github.ref == 'refs/heads/master' && 'cp39-* cp310-* cp311-*' || 'cp311-*' }}
```

Comment:
In the current state, aarch64 builds take long to build due to qemu. Adding another build version would be too much - therefore better remove 3.9.